metabase/metabase
# Always show metrics and models in menu

This is a follow up for https://github.com/metabase/metabase/pull/47277

### Description

After some discussion it was decided that we want to show the browse models and browse metrics link in the sidebar even when there are no models or metrics.

### How to verify


1. Make sure you delete all models and metrics
2. Both `models` and `metrics` should still show up in the sidebar and navigate to the correct pages
